cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/aggregation/AggregationDefinition.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.etl.api.aggregation;
+
+import io.cdap.cdap.api.annotation.Beta;
+import io.cdap.cdap.etl.api.relational.Expression;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Specifies how an aggregation should be executed.
+ */
+@Beta
+public abstract class AggregationDefinition {
+  private final List<Expression> groupByExpressions;
+  private final List<Expression> selectedExpressions;
+
+  /**
+   * Creates a default {@link AggregationDefinition} object with empty lists for group by and select expressions.
+   */
+  protected AggregationDefinition() {
+    groupByExpressions = Collections.emptyList();
+    selectedExpressions = Collections.emptyList();
```

Comment:
Use "selectExpression". Or even "resultExpressions"  as "select" is too tied to SQL and in fact it would define expressions that form the result Relation

Reply:
Changed to `selectExpression`. I believe we should stick to SQL nomenclature as everyone is familiar with it.